events/2022.03.Online/Architecture/Impls/ditto.html

```diff
@@ -0,0 +1,23 @@
+<div class="impl" id="impl-ditto">
+  <h4 id="impl-ditto-WoT">Eclipse Ditto: TD Exposer</h4>
+  <p>
+    A service that exposes Eclipse Ditto managed digital twins with Thing
+    Descriptions. More information can be found at
+    <https://www.eclipse.org/ditto/basic-wot-integration.html>
+  </p>
+  <p><b>Usage:</b> Service</p>
+  <p><b>Public Repository:</b> https://github.com/eclipse/ditto</p>
+  <p><b>Contributing Member Organizations:</b> Bosch.IO</p>
```

Comment:
```suggestion
  <p>
    A service that provides APIs for managing digital twins and interacting with the 
    abstracted devices. Fetches linked Thing Models of digital twins and provides instance 
    specific Thing Descriptions for them.
  </p>
  <p><b>Usage:</b> Service</p>
  <p><b>Public Repository:</b> https://github.com/eclipse/ditto</p>
  <p><b>Website:</b> https://www.eclipse.org/ditto/basic-wot-integration.html</p>
  <p><b>Contributing Member Organizations:</b> Bosch.IO</p>
```

I would suggest the above instead - would that be ok?